conformance/conformance.go

```diff
@@ -161,6 +175,7 @@ func DefaultOptions(t *testing.T) confsuite.ConformanceOptions {
 	for feature := range opts.ExemptFeatures {
 		opts.SupportedFeatures.Delete(feature)
 	}
+	t.Logf("CONFORMANCE.GO: Final opts.SupportedFeatures: %v", opts.SupportedFeatures.UnsortedList())
```

Comment:
same as above comment - I'd go without this log by default. maybe as a debug or remove it

Reply:
Done. ( removed )